src/main/java/data/workouts/WorkoutList.java

```diff
@@ -210,6 +210,22 @@ public void listWorkout() {
         logger.exiting(getClass().getName(), "listWorkout");
     }
 
+    /**
+     * Prints all workouts stored in workout list at once.
+     */
+    public void listAllWorkout() {
+        int index = 0;
+        System.out.println("Showing workouts " + (index + 1) + "-" + workoutsDisplayList.size()
+                + " of " + workoutsDisplayList.size() + ":");
+        while (index < workoutsDisplayList.size()) {
+            Workout workoutObject = getWorkoutFromIndexNum(index + 1);
+            System.out.println(index + 1 + ". " + workoutObject.toString());
+            index += 1;
+            assert (index <= workoutsDisplayList.size());
+        }
+        System.out.println("Showed all workouts in list");
+    }
+
     /**
      * Prints workouts stored in workout list.
```

Comment:
Perhaps could explain a bit further on this JavaDoc. E.g. prints workout in 20 workouts per page.

Reply:
Can you elaborate further? I do not quite understand what you mean? This listall feature will display all workouts in the workoutlist and the list feature will display at most 10 workouts at a time. [advance feature]